main/modes/picross/mode_picross.c

```diff
@@ -1090,29 +1371,41 @@ void picrossExitGame(void)
 // SAVING AND LOADING
```

Comment:
`main` does have functions now to read and write blobs of binary data in NVM. If you wanted to, you could make a struct with all the picross data and do a single read/write, rather than banking it into 32 bit ints. Probably cleaner, also a bit more work from where you are. Your call.
```C
bool readNvsBlob(const char* key, void* out_value, size_t* length);
bool writeNvsBlob(const char* key, const void* value, size_t length);
```

Reply:
oof, but done.

Humerously, the JSON file in the emulator serializes it way LESS efficiently now. An array of bools is now serialized as a string of 1s and 0s with each bool taking 2 spaces. Similar silliness with the 2d array of enums that is the level. I believe this is the emu JSON serializer, but "fyi" because perhaps I am just doing something wrong.